pkg/e2e/operators/cloudingress/certificate.go

```diff
@@ -22,18 +21,14 @@ var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
 			ingress1, _ := getingressController(h, "default")
 			originalCert = string(ingress1.Spec.DefaultCertificate.Name)
 			updateCertificate(h, "foo-bar")
-			time.Sleep(time.Duration(120) * time.Second)
 			ingress, _ := getingressController(h, "default")
-
+			time.Sleep(time.Duration(120) * time.Second)
```

Comment:
This sleep should also be removed and replaced with a `wait.PollImmediate` block that checks for the condition you're testing or times out.

As it stands, I don't think this sleep does anything useful.

Reply:
yes you're right! i was trying to remove all the sleep statements and missed this one !

Reply:
removed the sleep statement above.